test/FunctionalTests/LocalizationTests/CustomResourceManagerTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Threading.Tasks;
+using EntropyTests;
+using Microsoft.AspNetCore.Server.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+
+namespace FunctionalTests.LocalizationTests
+{
+    public class CustomResourceManagerTests
+    {
+        private const string SiteName = "Localization.CustomResourceManager";
+
+        [Theory]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, "http://localhost:9200")]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.Clr, RuntimeArchitecture.x86, "http://localhost:9201", Skip = "x86 not supported yet")]
```

Comment:
I believe so, yes. The rest of them throughout the repo are commented out, I've already made #157 to track restoring (or at least skipping instead of commenting out) these.